ci/run-check.sh

```diff
@@ -41,26 +41,26 @@ case $TARGET in
   docs-build)
     RUSTDOCFLAGS="-D warnings" cargo doc --all --no-deps
     ;;
-  
+
   try-runtime)
-  if [ "$1" == "altair" ]; then
-    echo "Running try-runtime for altair"
+    cargo build -p centrifuge-chain --release --features try-runtime
+    if [ "$1" == "altair" ]; then
+      echo "Running try-runtime for altair"
       RUST_LOG=runtime=trace,try-runtime::cli=trace,executor=trace \
-      cargo run --release --features try-runtime try-runtime \
+      try-runtime \
```

Comment:
This will break the `try-runtime` CI. Since you pinged @gpmayorga already and `try-runtime` being optional, I am fine with leaving this as is.

Reply:
> This will break the try-runtime CI?

Mmm... you said that because we're not installing `try-runtime` binary, right? Probably. I've tested locally but I have the tool 😆

Reply:
Since we are not really utilizing the CI right now, totally fine with this.